db/migrate/20220324071219_create_posts.rb

```diff
@@ -0,0 +1,13 @@
+class CreatePosts < ActiveRecord::Migration[7.0]
+  def change
+    create_table :posts do |t|
+      t.references :user, null: false, foreign_key: true, index: true
+      t.string :title
+      t.string :text
```

Comment:
- [ ] Please use a `text` instead of a `string` for the post text because it may be longer